backend/open_webui/main.py

```diff
@@ -195,6 +196,9 @@ async def get_response(self, path: str, scope):
 
 @asynccontextmanager
 async def lifespan(app: FastAPI):
+    start_logger()
+    run_migrations()
```

Comment:
`run_migration` is invoked from the config.py, this should be removed.

Reply:
Sure I should remove this in this PR? It was already there before, the reason it showed up in the PR is that I added the `start_logger` before `run_migrations() call

Reply:
it was originally in there but got removed from main due to several issues.